src/components/SingleVideo/singlevideo.css

```diff
@@ -90,7 +90,47 @@
             "details"
             "suggested";
     }
-    .video-detail {
-        display: none;
+
+    .video-container {
+        min-height: 22rem;
+    }
+
+    .video-info {
+        width: 100vw;
+    }
+
+    .add-comment input {
+        max-width: 50%;
+    }
+}
+
+@media screen and (max-width:820px) {
+    .single-video-grid {
+        display: grid;
+        grid-template-columns: 1fr;
```

Comment:
in media queries if properties are same among all queries you can skip to write those properties instead write those in one media query it will applies in all media queries like here display **grid** and gtc:**1fr**